lib/smart_answer_flows/minimum-wage-calculator-employers.rb

```diff
@@ -36,10 +36,6 @@ def define
 
       # Q3
       value_question :how_old_are_you?, parse: Integer do
-        precalculate :age_title do
-          "How old are you?"
```

Comment:
Should there be a corresponding inline-into-view change for this?

Reply:
Oddly not - [the view doesn't use this value at all](https://github.com/alphagov/smart-answers/blob/bc167d2e94674549b93b31e8881b9c0a9bd7892a/lib/smart_answer_flows/minimum-wage-calculator-employers/questions/how_old_are_you.erb#L2).

But this change snuck in here - I'm going to remove it and re-raise apart of a PR to refactor the minimum wage calculator for employers flow